src/main/java/my/test/email/CreateDriver.java

```diff
@@ -0,0 +1,17 @@
+package my.test.email;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class CreateDriver {
```

Comment:
Make correct singleton:
1. private static instance of singleton class
2. private constructor
3. public static getInstance() method

WebDriver instance will be non-static. How to get WebDriver:
`YourSingleton.getInstance().getDriver();`